src/routes/Home/index.tsx

```diff
@@ -1,14 +1,12 @@
 import { ReactElement } from 'react'
 import styled from 'styled-components'
-import { Breadcrumb, BreadcrumbElement, Menu } from '@gnosis.pm/safe-react-components'
 
 import Page from 'src/components/layout/Page'
-import Row from 'src/components/layout/Row'
-import Col from 'src/components/layout/Col'
 import PendingTxsList from 'src/components/Dashboard/PendingTxs/PendingTxsList'
-
 import AddSafeWidget from 'src/components/Dashboard/AddSafe'
 import CreateSafeWidget from 'src/components/Dashboard/CreateSafe'
+import SafeApps from 'src/components/Dashboard/SafeApps/Grid'
```

Comment:
Can we either rename the component to `SafeApps` or import it as `Grid`?

Reply:
changed the import name to SafeAppsGrid